packages/contract-addresses/addresses.json

```diff
@@ -225,7 +225,7 @@
             "wethTransformer": "0x02ce7af6520e2862f961f5d7eda746642865179c",
             "payTakerTransformer": "0xa6c3ca183a67fcb4299fb4199c12ca74874ca489",
             "affiliateFeeTransformer": "0x3102aea537ecb6f164550b094663c82a8c53a972",
-            "fillQuoteTransformer": "0xfead4c05220a8fddc74008ae43199badc3becaf9",
+            "fillQuoteTransformer": "0xd140adb61d4e3e3978d4f32ac6b92240ff6e3a6e",
```

Comment:
this doesn't look correct.

Reply:
https://optimistic.etherscan.io/tx/0xa5ae6ec17c5cff0b4f7ecf6c0b107a7d401c3ec68170692a75d2fd6b8480ce7b#eventlog
Heres my transaction that deployed the FQT. I'm not entirely sure why but it doesn't appear to be correct in optimistic etherscan. However, I yarn linked contract-addresses against the API and I was able to fill through this FQT.

Reply:
discussed on slack